search/indexing_api.py

```diff
@@ -191,15 +189,6 @@
 }
 
 
-BOOTCAMP_OBJECT_TYPE = {
-    **LEARNING_RESOURCE_TYPE,
-    "id": {"type": "long"},
-    "course_id": {"type": "keyword"},
-    "full_description": ENGLISH_TEXT_FIELD,
-    "location": {"type": "keyword"},
```

Comment:
Does "location" need to be added to COURSE_OBJECT_TYPE? Or is it just not used anywhere in the search UI?

Reply:
I don't think it's used but will add it anyway just in case